tests/Integration/NotifierTest.php

```diff
@@ -105,23 +106,49 @@ public function test_sendAlertsPerEmailToRecipient()
 
     public function test_sendAlertsPerEmailToRecipient_shouldUseDifferentSubjectDependingOnPeriod()
     {
-        $this->assertDateInSubject('week', 'week December 21 - 27, 2009');
-        $this->assertDateInSubject('day', 'Thursday, December 31, 2009');
-        $this->assertDateInSubject('month', 'December 2009');
+        $this->assertDateInSubject('week', $this->getMockMail('week')->getSubject());
```

Comment:
I also don't understand the changes made in the tests. It seems to no longer test what it was supposed to test. Are these changes in the tests needed?